### PR TITLE
nala: add page

### PR DESCRIPTION
**Pull request by rafay99-epic**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/nala.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+
+# nala
+
+> Debian and Ubuntu package management utility.
```
Comment (kbdharun) [inline pages/linux/nala.md:4]:
```suggestion
> Package management utility for Debian and Ubuntu.
```
How about this?

File: pages/linux/nala.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+> Search for packages using `nala search`.
+> More information: <https://gitlab.com/volian/nala>
```
Comment (kbdharun) [inline pages/linux/nala.md:6]:
```suggestion
> More information: <https://gitlab.com/volian/nala>.
```

File: pages/linux/nala.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+- Remove all packages that are no longer needed:
+
+`nala autoremove`
+
+- Upgrade installed packages (like `upgrade`), but remove obsolete packages and install additional packages to meet new dependencies:
+
+`nala dist-upgrade`
+
+- Nala will then go get all the mirrors from the respective master list. Once done we test the latency and score each mirror. Nala will choose the fastest 3 mirrors (configurable) and write them to a file.
```
Comment (kbdharun) [inline pages/linux/nala.md:40]:
```suggestion
- Fetch mirrors from the master list for your distribution (Nala will choose the fastest 3 configurable mirrors by testing latency and write them to a file):
```

File: pages/linux/nala.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+- Remove a package and its configuration files:
+
+`nala purge {{package}}`
+
+- Upgrade all installed packages to their newest available versions:
+
+`nala upgrade`
+
+- Clean the local repository - removing package files (`.deb`) from interrupted downloads that can no longer be downloaded:
```
Comment (kbdharun) [inline pages/linux/nala.md:28]:
```suggestion
- Clean the local repository by removing package files (`.deb`) from interrupted downloads that can no longer be downloaded:
```

File: pages/linux/nala.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+`nala autoclean`
+
+- Remove all packages that are no longer needed:
+
+`nala autoremove`
+
+- Upgrade installed packages (like `upgrade`), but remove obsolete packages and install additional packages to meet new dependencies:
```
Comment (kbdharun) [inline pages/linux/nala.md:36]:
```suggestion
- Upgrade installed packages, remove obsolete packages and install additional packages to meet new dependencies:
```

File: pages/linux/nala.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+- Nala will then go get all the mirrors from the respective master list. Once done we test the latency and score each mirror. Nala will choose the fastest 3 mirrors (configurable) and write them to a file.
+
+`nala fetch`
+
+- History
+
+`nala history`
+
+
+
```
Comment (kbdharun) [inline pages/linux/nala.md:49]:
```suggestion
```

File: pages/linux/nala.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+- Upgrade installed packages (like `upgrade`), but remove obsolete packages and install additional packages to meet new dependencies:
+
+`nala dist-upgrade`
+
+- Nala will then go get all the mirrors from the respective master list. Once done we test the latency and score each mirror. Nala will choose the fastest 3 mirrors (configurable) and write them to a file.
+
+`nala fetch`
+
+- History
```
Comment (kbdharun) [inline pages/linux/nala.md:44]:
```suggestion
- Print history of all transactions:
```

File: pages/linux/nala.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
...
+
+- Upgrade all installed packages to their newest available versions:
+
+`nala upgrade`
+
+- Clean the local repository - removing package files (`.deb`) from interrupted downloads that can no longer be downloaded:
+
+`nala autoclean`
+
+- Remove all packages that are no longer needed:
```
Comment (kbdharun) [inline pages/linux/nala.md:32]:
```suggestion
- Remove all unused packages and dependencies:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution and welcome to tldr. I have some suggestions for this page.

File: pages/linux/nala.md [hunk 9]
```diff
@@ -0,0 +1,49 @@
+
```
Comment (kbdharun) [inline pages/linux/nala.md:1]:
```suggestion
```

**Review by kbdharun — CHANGES_REQUESTED** [review]
Our pages can only hold a maximum of 8 examples, feel free to update the page in order to have a successful CI check.

**Comment by rafay99-epic** [comment]
Sure, I will add some examples

**Comment by EmilySeville7cfg** [comment]
@rafay99-epic, if u need unlimited amount of examples u can use my TlDr-like project. [Here](https://github.com/command-line-interface-pages/cli-pages/pull/47) is an example how it looks.

**Comment by rafay99-epic** [comment]
Hey thanks I will look into it. @EmilySeville7cfg

**Comment by rafay99-epic** [comment]
> [CLA assistant check](https://cla-assistant.io/tldr-pages/tldr?pullRequest=9890) All committers have signed the CLA.

Done, signature this using my GitHub Account

**Comment by rafay99-epic** [comment]
> The [build](https://github.com/tldr-pages/tldr/actions/runs/4318045983) for this PR failed with the following error(s):
> 
> ```
> pages/linux/nala.md:5: TLDR016 Label for information link should be spelled exactly `More information: `
> pages/linux/nala.md:22: TLDR014 Page should contain no trailing whitespace
> pages/linux/nala.md:25: TLDR014 Page should contain no trailing whitespace
> pages/linux/nala.md:0: TLDR019 Page should only include a maximum of 8 examples
> ```
> 
> Please fix the error(s) and push again.

@kbdharun  Brother, Can you explain this error? I can't seem to understand this.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by mfrw — APPROVED** [review]
LGTM Thanks for your contribution @rafay99-epic

**Comment by rafay99-epic** [comment]
> LGTM Thanks for your contribution @rafay99-epic

Happy to Contribute, and make this application more powerful and awesome.

**Pull request merged**